tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "ES2016",
```

Comment:
Will bump this to [ES6](https://www.typescriptlang.org/tsconfig#target) at a later date